src/oic/oauth2/__init__.py

```diff
@@ -11,6 +11,7 @@
 from urllib.parse import urlparse
 
 from jwkest import b64e
+from requests import Session
```

Comment:
If I see correctly, this is going to be used only for the Client code?

Reply:
removed and moved request_session to settings